### PR TITLE
FIX write_labels_to_annot():  List of RH labels only

### PR DESCRIPTION
**Pull request by christianbrodbeck**
If a user tried to save an `annot` containing only RH-labels, `ctab_rgb` was not defined in the first (LH) iteration

### CODE REVIEW

File: mne/label.py [hunk 1]
```diff
@@ -1581,12 +1581,9 @@ def _get_annot_fname(annot_fname, subject, hemi, parc, subjects_dir):
...
 
-        annot_fname = list()
-        for hemi in hemis:
-            subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
-            fname = op.join(subjects_dir, subject, 'label',
-                            '%s.%s.annot' % (hemi, parc))
-            annot_fname.append(fname)
+        subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+        dst = op.join(subjects_dir, subject, 'label', '%%s.%s.annot' % parc)
+        annot_fname = [dst % hemi for hemi in hemis]
```
Comment (christianbrodbeck) [inline mne/label.py]: This is just a stylistic change

File: mne/label.py [hunk 2]
```diff
@@ -1800,6 +1797,7 @@ def write_labels_to_annot(labels, subject=None, parc=None, overwrite=False,
 
         if n_hemi_labels == 0:
             ctab = np.empty((0, 4), dtype=np.int32)
+            ctab_rgb = ctab[:, :3]
```
Comment (christianbrodbeck) [inline mne/label.py]: this is the bug fix

**Comment by Eric89GXL** [comment]
LGTM, +1 for merge after PEP8 fix

https://travis-ci.org/mne-tools/mne-python/jobs/64736715#L2340

**Comment by agramfort** [comment]
+1

**Comment by agramfort** [comment]
thanks @christianbrodbeck

**Pull request merged**